src/content/translations/zh-tw/nft/index.md

```diff
@@ -0,0 +1,447 @@
+---
+title: 非同質化代幣 (NFT)
+description: 以太坊生態系非同質化代幣概要
+lang: zh-tw
+template: use-cases
+emoji: ":frame_with_picture:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/infrastructure_transparent.png
```

Comment:
@pettinarip for English pages ../../assets/~ is the correct path, but we require two more levels (i.e. ../../../../assets/~) for files on translated pages. 

Didn't we implement a fix for image paths in the frontmatter? Or was that only for images within markdown?

Reply:
No, I didn't do that in the end. I didn't know how to do it on those images so I leave it. But I guess it is something we can investigate further if it is worth it.